### ISSUE TITLE
Branch features/corr

### ISSUE THREAD

**Issue by ulfmueller** [openego/eTraGo#150]
https://github.com/openego/eTraGo/tree/features/corr

@maltesc are you using this branch? Is it one to keep or anything we can merge from there into dev?

**Comment by maltesc** [comment]
> are you using this branch? 

Yes, I use this branch for everything related to my thesis. If this is not OK, I can switch to a private repo.

> Is it one to keep or anything we can merge from there into dev?

Yes, I think there is plenty of code that can be used.

**Comment by ulfmueller** [comment]
It is totally ok. Keep working on this branch. I was just cleaning up a bit...

**Comment by ulfmueller** [comment]
After your thesis we can check what code we can include how into the mainstream development. I will leave this issue open in order to have a reminder for this task.
